subxt/src/backend/unstable/rpc_methods.rs

```diff
@@ -288,6 +288,28 @@ impl<T: Config> UnstableRpcMethods<T> {
 
         Ok(TransactionSubscription { sub, done: false })
     }
+
+    /// Broadcast the transaction on the p2p network until the
+    ///[`Self::transaction_unstable_stop`] is called.
+    ///
+    /// Returns an operation ID that can be used to stop the broadcasting process.
+    /// Returns `None` if the server cannot handle the request at the moment.
+    pub async fn transaction_unstable_broadcast(&self, tx: &[u8]) -> Result<Option<String>, Error> {
```

Comment:
Has the PR to rename to `v1` merged in Substrate already? :)

Reply:
yupp, https://github.com/paritytech/polkadot-sdk/blob/master/substrate/client/rpc-spec-v2/src/transaction/api.rs#L50